code/extensions/che-commands/package.json

```diff
@@ -23,7 +23,6 @@
   },
   "main": "./out/extension.js",
   "scripts": {
-    "prepare": "yarn compile && yarn test",
```

Comment:
so, the solution is - just turn off tests at all?

Reply:
The short answer: to fix the failed downstream build we need to make the dependencies installation work again - as it used to be a few weeks ago.

@RomanNikitenko I see your concern. Let me explain the problem from a bit different perspective with more details.
We have in-progress work on the new downstream build approach - with Cachito. It requires running the dependencies installation (calling `yarn install`) for each VS Code sub-module separately. Once the dependencies are installed for each sub-module, we can run the `gulp` build as the `gulp` binary already installed. It used to work as expected until we introduced [this change](https://github.com/che-incubator/che-code/commit/319c20aa848e32c2c5e49b1ccbac2acd1b4c128d) recently. The mentioned change means that there's no more a way to just install the dependencies solely for the `che-commands` sub-package. Moreover, unlike other sub-packages, calling `yarn install` for `che-commands` now triggers all three steps:
- dependencies installation;
- actual build;
- running tests.

In other words, when one tries just to install the dependencies for the `che-commands` sub-module, `yarn install` command always fails.

So, this PR just reverts the regression introduced by the recent change to fix the failed downstream build.
Once we migrate to the new downstream build approach we will be able to detect such problems much faster. As our downstream build job will comment on the corresponding Che-Code PR immediately that the downstream build has failed.

To separate the broken extension's dependencies installation problem from the problem of automating running the tests (instead of calling `yarn test` manually) we can create a separate issue to discuss the ideas for the second problem. Will it work well for you, @RomanNikitenko?

Reply:
> to fix the failed downstream build we need to make the dependencies installation work again - as it used to be a few weeks ago.

The corresponding jobs are green for the downstream build:
![image](https://github.com/che-incubator/che-code/assets/5676062/862dc966-e671-47c3-ae47-4675f2b13c8a)


So, I guess, it's more about:
> We have in-progress work on the new downstream build approach - with Cachito. It requires running the dependencies installation (calling yarn install) for each VS Code sub-module separately.

Correct?

I understand that running tests should be moved on the another stage, but, I do believe we should not just turn off them. It's a very strange situation that **we have tests** for our extensions (btw, not only for the `che-commands`) and **we don't run them at all**: https://github.com/che-incubator/che-code/pull/227#issuecomment-1548387927.

About a separate issue to turn on tests for all our extensions - yes - it works for me, I hope the issue will get a high priority, otherwise it means that our tests are going to have quite long holiday :-)

Reply:
> > So, I guess, it's more about:
> 
> We have in-progress work on the new downstream build approach - with Cachito. It requires running the dependencies installation (calling yarn install) for each VS Code sub-module separately.
> 
> Correct?

Correct.